### DISCUSSION TITLE
How to credit pillow or PIL in my project?

### DISCUSSION THREAD

**Discussion by BhagyaJyoti22006** [python-pillow/Pillow#8320] [Q&A] [answered]
I have a project. It will be free and open source. How shall I credit pil/pillow in the project.

**Comment by hugovk** [comment, marked as answer]
No credit needed as long as the [licence](https://github.com/python-pillow/Pillow/blob/main/LICENSE) is respected, but feel free to link to https://github.com/python-pillow/Pillow or https://python-pillow.org if you like.

For formal citations, you may use this Digital Object Identifier (DOI):

https://zenodo.org/doi/10.5281/zenodo.596518

Thanks!

**Reply by BhagyaJyoti22006** [reply]
Thanks. I will put image Processing by Pillow (Pil)
